tests/util/test_createPREMISEventXML.py

```diff
@@ -0,0 +1,447 @@
+    kwarg.
+    """
+    premis_args.pop('eventIdentifier')
+    util.createPREMISEventXML(**premis_args)
+
+
+def test_with_empty_linkObjectList_arguments(premis_args):
+    """
+    Check the function is successful without the linkObjectList
+    kwarg.
+    """
+    premis_args.pop('linkObjectList')
+    util.createPREMISEventXML(**premis_args)
+
+
+def test_with_empty_eventDate_arguments(premis_args):
+    """
+    Check the function is successful without the eventDate
+    kwarg.
+    """
+    premis_args.pop('eventDate')
+    util.createPREMISEventXML(**premis_args)
+
+
+def test_has_event_element():
+    """
+    Verify the root element is `event`.
+    """
+    premis = util.createPREMISEventXML(None, None, None, None)
+
+    event = premis.xpath(
+        '/p:event',
+        namespaces={'p': util.PREMIS_NAMESPACE}
+    )
+    assert len(event) == 1
+
+
+def test_eventType_element_text():
+    """
+    Verify the eventType positional argument yields an eventType
+    element with the text from the argument.
+    """
+    event_type_text = 'Fake Event Type'
+    premis = util.createPREMISEventXML(event_type_text, None, None, None)
+
+    eventType = premis.xpath(
+        '/p:event/p:eventType',
+        namespaces={'p': util.PREMIS_NAMESPACE}
+    )
+    assert len(eventType) == 1
+    assert eventType[0].text == event_type_text
+
+
+def test_eventType_element_text_is_none():
+    """
+    Verify the eventType positional argument yields an eventType
+    element with no text.
+    """
+    premis = util.createPREMISEventXML(None, None, None, None)
+
+    eventType = premis.xpath(
+        '/p:event/p:eventType',
+        namespaces={'p': util.PREMIS_NAMESPACE}
+    )
+    assert len(eventType) == 1
+    assert eventType[0].text is None
+
+
+def test_has_eventIdentifier_element():
+    """
+    Check that the eventIdentifier tree is present.
+    """
+    premis = util.createPREMISEventXML(None, None, None, None)
+    eventIdentifier = premis.xpath(
+        '/p:event/p:eventIdentifier',
+        namespaces={'p': util.PREMIS_NAMESPACE}
+    )
+    assert len(eventIdentifier) == 1
+    assert len(eventIdentifier[0].getchildren()) == 2
+
+
+def test_eventIdentifierValue_element_text():
+    """
+    Verify the eventIdentifier kwarg yields an eventIdentifier element with
+    the text from the argument.
+    """
+    eventIdentifier = 'ABC'
+    premis = util.createPREMISEventXML(None, None, None, None,
+                                       eventIdentifier=eventIdentifier)
+    eventIdentifierValue = premis.xpath(
+        '/p:event/p:eventIdentifier/p:eventIdentifierValue',
+        namespaces={'p': util.PREMIS_NAMESPACE}
+    )
+    assert len(eventIdentifierValue) == 1
+    assert eventIdentifierValue[0].text == eventIdentifier
+
+
+def test_eventIdentifierValue_element_default_text():
+    """
+    Verify the eventIdentifier kwarg yields an eventIdentifier element with
+    the default text when it is falsy.
```

Comment:
I personally think 'falsy' is a great word, but I don't think the world is ready for it just yet.